app/src/main/java/org/p2p/wallet/utils/NavigationExtensions.kt

```diff
@@ -136,6 +137,27 @@ fun Fragment.replaceFragment(
     }
 }
 
+fun Fragment.replaceFragmentForResult(
```

Comment:
Missing `Activity`s? :D 
I still haven't decided will we use this or not, because we don't have selection fragment yet. But I thing it's a good out-of-box option for getting results from another fragment.